docs/competition/matches/match-scoring.md

```diff
@@ -11,47 +11,76 @@ Any roles related to scoring matches go here.
+
+Once you have finished scoring, you must drop your completed score sheet at the score entry desk which will be near the arenas. You must return promptly to the arena in case you need to be found for further scoring questions, and begin resetting the arena for the next match.
+
+### Teams touching robots
+
+Teams must not be allowed in the arena during a match. It is the responsibility of the team members to place their robot in the start zone before the match starts. If a robot moves off before the match starts (aside from minor servo-resetting arm movements) teams are allowed to switch off their robot, reset it and start again. However, if the mis-started robot touches any arena prop then they should be immediately disqualified (there is a checkbox on the score sheet). Once the robot start button has been pressed and the match has started, teams are not allowed to interact with their robots until after scoring has taken place.
+
+### Turning off robots
+
+Be sure to turn off robots if they continue moving after the end of a match. Make sure you know where the off button is for all robots before the matches.
+
+There are two situations where robots should be turned off before the end of a match:
+- If the team members indicate that they wish to do so (by "throwing in the towel"). By doing so, the team forfeits the match, which should be recorded in the score sheets as a disqualification.
+- When a robot is causing **significant** damage to itself or another robot, i.e. Mashing into another robot, stopping it from being able to function, or dragging it around without being able to be freed, or overheating a motor by stalling against a wall. It's up to you to decide which robot should be switched off, but their match shouldn't be forfeited. If any damage seems intentional, please report it to the head judge. A checklist for robot damage is available below.
 
 If you have any questions you may first ask the head match scorer and secondly the judge.
 
 ### Template Arena Setup Checklist
 
-1. Tokens in the arena aren't damaged. If they are, swap them out with the spares.
+1. Props (e.g: tokens) in the arena aren't damaged. If they are, swap them out with the spares.
 2. The tokens in the arena are intended for that arena -- this can be established by reading the grey text printed on each marker on the token.
    If there is more than one arena then the markers on the sides of any tokens will have the name of the arena as part of the text in the bottom left of the marker.
 3. There must be the correct number of tokens of each type in the arena. The precise configuration depends on the game, though the per-year version of this checklist should be explicit about what's needed (and not rely on volunteers remembering the rules)
```

Comment:
Just highlighting that we should also prep a per-year checklist for the arena. Likely not part of this PR, but should work out who's going to do that. @Adimote did you have in mind to do that? Alternatively, I know that @sedders123 has taken on a lot of the arena related stuff and might already have looked at this?

Reply:
Ack, have created https://github.com/srobo/tasks/issues/252 for this